packages/core/src/lib/icon/icon.svelte

```diff
@@ -41,23 +41,14 @@ export let size: Size = 'base';
 /** Additional CSS classes to pass to the svg. */
 let extraClasses: cx.Argument = '';
 export { extraClasses as cx };
-
-const hasNameProperty = Object.hasOwn(paths, name);
 </script>
 
 <!--
   Accessibility approach for icon svgs taken from:
   https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label
 -->
 <svg
-  class={cx(
-    'text-gray-6',
-    sizes[size],
-    {
-      hidden: !hasNameProperty,
```

Comment:
Why the removal of `hidden` here?

Reply:
It had been annoying me and it's tripped folks up in the past. If a `name` prop is passed in to the `Icon` that isn't in `iconData`, that is a bug. Adding a `hidden` class to the icon when `name` is invalid feels like pretending that everything is ok